docs/bir-spec/src/main/resources/kaitai/bir.ksy

```diff
@@ -1417,6 +1417,11 @@ types:
         type: s4
       - id: lhs_operand
         type: operand
+      - id: has_typedesc_operand
+        type: s1
+      - id: typedesc_operand
+        type: operand
+        if: has_typedesc_operand == 1
       - id: size_operand
```

Comment:
In this `bir.ksy` file there is a minor typo. Shall we change all the occurrences of `golbal` to `global`